api/models/Comment.js

```diff
@@ -45,6 +57,33 @@ module.exports = bookshelf.Model.extend(Object.assign({
     return this.belongsTo(Comment).where('comments.active', true)
   },
 
+  reaction: function (userId, data) {
```

Comment:
yeah more and more i think it would be good to change this to be 'react' or 'reactTo' or 'addReaction'